### PR TITLE
add normal user for access control checking

### PR DESCRIPTION
**Pull request by richm**
The previous commit did had some leftover debugging code.
@jcantrill @ewolinetz PTAL
[test]

### CODE REVIEW

File: hack/testing/logging.sh [hunk 1]
```diff
@@ -142,7 +142,7 @@ os::cmd::expect_success 'oadm router'
 ######### logging specific code starts here ####################
 
 os::cmd::expect_success "oadm new-project logging --node-selector=''"
-os::cmd::expect_success "oc project logging"
+os::cmd::expect_success "oc project logging > /dev/null"
```
Comment (ewolinetz) [inline hack/testing/logging.sh]: just out of curiosity is this just to suppress the output that we may already be on this project? e.g. this has no impact on the expect_success

**Comment by jcantrill** [comment]
LGTM

File: hack/testing/logging.sh (hunk 1 above)
Comment (richm) [inline hack/testing/logging.sh]: @ewolinetz correct - it just reduces the noise in our tests - has no impact on expect_success.

**Comment by richm** [comment]
[merge]

**Pull request merged**